src/vs/editor/common/editorCommon.ts

```diff
@@ -4164,6 +4167,33 @@ export var EventType = {
 };
 
 /**
+ * Positions on the view
+ */
+export const ViewPosition = {
+	LineStart: 'lineStart',
+	LineFirstNonWhitespaceCharacter: 'lineFirstNonWhitespaceCharacter',
```

Comment:
@alexandrudima We referred at other places and we do not use 'soft' word for beginning character. We say startCharacter and firstNonWhitespaceCharacter. Hence we used LineStart and LineFirstNonWhitespaceCharacter and I think they are easy to understand.